HOWTO-make-a-release.md

```diff
@@ -0,0 +1,400 @@
+
+# Post-publishing tasks
+
+## Check the website
+
+Verify that the release notes, which are built from the CHANGES file in the
+release, have been updated.  This is done automatically by the commit-hook,
+but if you see a problem, try the following steps on `dev.openssl.org`:
+
+    cd /var/www/openssl
+    sudo -u openssl -H make relupd
+    sudo -u openssl -H ./bin/purge-one-hour
+
+Wait for a while for the Akamai flush to work (normally within a few minutes).
+Have a look at the website and news announcement at:
+
+-   <https://www.openssl.org/>
+-   <https://www.openssl.org/news/>
+
+Check the download page has updated properly:
+
+-   <https://www.openssl.org/source/>
+
+Check the notes look sensible at:
+
+-   <https://www.openssl.org/news/newslog.html>
+
+Also check the notes here:
+
+-   <https://www.openssl.org/news/openssl-1.0.2-notes.html>
+-   <https://www.openssl.org/news/openssl-1.1.0-notes.html>
+-   <https://www.openssl.org/news/openssl-1.1.1-notes.html>
+
+## Send the announcement mail
+
+Send out the announcements.  Generic release announcement messages will be
+created automatically by the build script and the commands you need to use
+to send them were displayed when you executed do-release.pl above.
+These should normally be sent from the openssl account.  These are sent to
+openssl-users, openssl-project, and openssl-announce.
+
+If do-release.pl was used with `--move` be sure to move the announcement
+text files away from the staging directory after they have been sent.  This
+is done as follows (with VERSION replaced with the version of OpenSSL to
+announce):
+
+    sudo -u openssl \
+        mutt -s "OpenSSL version VERSION published" \
+            openssl-project openssl-users openssl-announce \
+            < /home/openssl/dist/new/openssl-VERSION.txt.asc
+    sudo -u openssl \
+        mv ~openssl/dist/new/openssl-VERSION.txt.asc ~openssl/dist/old
+
+## Send out the Security Advisory
+
+*The secadv file mentioned in this section is the Security Advisory
+that you copied into the web repo, up in the section
+[Update the website locally](#update-the-website-locally)*
+ 
+*This section is only applicable if this is a security release*
+
+Start with signing the Security Advisory as yourself:
+
+    gpg --clearsign secadv_FILENAME.txt
+
+Then copy the result to the temporary directory on dev.openssl.org:
+
+    scp secadv_FILENAME.txt.asc dev.openssl.org:/tmp
+
+To finish, log in on dev.openssl.org and send the signed Security
+Advisory by email as the openssl user, and the remove it:
+
+    sudo -u openssl mutt -s "OpenSSL Security Advisory" \
+            openssl-project openssl-users openssl-announce \
+            </tmp/secadv_FILENAME.txt.asc
+    rm /tmp/secadv_FILENAME.txt.asc
+
+Approve the openssl-announce email.  Go to
+<https://mta.openssl.org/mailman/admindb/openssl-announce>
+and approve the messages.
+The administration password needed for approval is held in /opt/mailman/README
```

Comment:
Do we really want the location of this to be public? I mean it already is now....but I'm not sure we should keep it that way.

Reply:
Ok.  Shall I just drop this line, then?

Reply:
I think so

Reply:
Done